src/main/java/se/iths/SpringbootGroupProject/entities/Message.java

```diff
@@ -18,8 +18,18 @@ public class Message {
 
     private String title;
     private String messageBody;
-    private boolean isPrivateMessage;
+    private boolean privateMessage;
+    @ManyToOne(fetch = FetchType.EAGER)
```

Comment:
Isn't ManyToOne is eager by default?

Reply:
wait.. is this because of Hibernate using lazy fetching?

Reply:
> wait.. is this because of Hibernate using lazy fetching?

Correct. We migh lose some performance but we do not need to load data and add more lines of code.